src/scripts/main.js

```diff
@@ -1 +1,22 @@
 'use strict';
+
+const link = document.querySelectorAll('.list-item__link');
+
+
+document.addEventListener('DOMContentLoaded', () => {
+  const bigImg = document.querySelector('#largeImg');
+  const images = document.querySelectorAll('.gallery__img.gallery__thumb');
+  const link = document.querySelectorAll('.list-item__link');
```

Comment:
The variable 'link' is redeclared here. Since 'link' is already declared in the outer scope, you should remove the 'const' keyword to avoid redeclaration and potential confusion.